MaterialLib/TwoPhaseModels/CreateTwoPhaseFlowMaterialProperties.cpp

```diff
@@ -24,75 +26,114 @@ namespace MaterialLib
 {
 namespace TwoPhaseFlowWithPP
 {
-std::unique_ptr<TwoPhaseFlowWithPPMaterialProperties>
-CreateTwoPhaseFlowMaterialProperties(
+std::tuple<std::unique_ptr<TwoPhaseFlowWithPPMaterialProperties>,
+           BaseLib::ConfigTree>
+createTwoPhaseFlowMaterialProperties(
     BaseLib::ConfigTree const& config,
-    bool const has_material_ids,
-    MeshLib::PropertyVector<int> const& material_ids)
+    boost::optional<MeshLib::PropertyVector<int> const&>
+        material_ids)
 {
     DBUG("Reading material properties of two-phase flow process.");
 
-    //! \ogs_file_param{prj__processes__process__TWOPHASE_FLOW_PP__material_property__fluid}
-    auto const& fluid_config = config.getConfigSubtree("fluid");
+    //! \ogs_file_param{material__twophase_flow_pp__material_property__fluid}
```

Comment:
I think this property class is for two-phase flow process. It is not only for the PP scheme, but also to PS and other scheme. Therefore I suggest to drop that `pp`.